### DISCUSSION TITLE
cross-platform shortcuts

### DISCUSSION THREAD

**Discussion by noahsbrom** [RobertWHurst/Keystrokes#30] [Q&A]
Does keystrokes support something similar to Mousetrap's 'mod', which is a generic helper for cross-platform shortcuts? In other words, is there a way to map 'control+s' on Windows and 'meta+s' on Mac to the same shortcut, such that 'control+s' on Mac does NOT trigger the shortcut?

**Comment by RobertWHurst** [comment]
My suggestion would be to calculate this ahead of creating your bindings.

```ts
const isMac = navigation.userAgent.toLowerCase().includes('mac')

const modKey = isMac ? 'meta' : 'control'

bindKeyCombo(`${modKey} > k`, () => console.log('do something awesome'))
bindKeyCombo(`${modKey} > j`, () => console.log('do something else awesome'))
```
